### ISSUE TITLE
Enabling FXS port based analog telephony with chan_lantiq and asterisk13

### ISSUE THREAD

**Issue by ORippler** [Quallenauge/Easybox-904-XDSL#17]
Hey,

I want to use the FXS plug of the Easybox as outlined [here](https://openwrt.org/docs/guide-user/services/voip/asterisk/chan-lantiq).
However, recommended installation via `opkg` fails with:

`opkg_install_cmd: Cannot install package asterisk15.`

so I guess I will have to build my own image. Do you have any experience regarding FXS telephonie and the EasyBox? Especially, since on the OpenWRT page it states that the [analog telephone line](https://openwrt.org/toh/astoria/arcadyan_astoria_easybox_904xdsl_r01?s%5B%5D=arcadyan&s%5B%5D=astoria&s%5B%5D=easybox&s%5B%5D=904xdsl#installing_lede) should work and it's referenced in #6.

I installed the latest snapshot (SNAPSHOT, r8430-b904437c2d) via the fullimage.img method.

Best,

**Comment by ORippler** [comment]
Is the newest snapshot maybe compiled without asterisk? 

I guess I will try another day, and start [here](https://forum.openwrt.org/t/voip-configuration-for-asterisk13-pjsip-chan-lantiq-and-vodafone-germany/9470) for configuration

**Comment by ORippler** [comment]
Just a quick update: I managed to install asterisk and register my VOIP-provider.
However, I cannot seem to properly setup a working dialplan/provider-context:
 * For incoming calls, i get an authentication failure (`chan_sip.c:25611 handle_request_invite: Failed to authenticate device`). I suspect the fault here may lie in my `sip.conf`
 * For outgoing calls, I get an `app_stack.c:594 gosub_exec: Attempt to reach a non-existent destination for Gosub: (Context:tel1_out, Extension:out_sipgate, Priority:2)`. When upping verbosity/debug level, I see that the phone is `circuit-busy`

For the configuration, I used this post at [IP-phone](https://www.ip-phone-forum.de/threads/o2-voip-mit-asterisk.299004/) for reference regarding SIP configuration, and the [chan_lantiq post on OpenWRT](https://openwrt.org/docs/guide-user/services/voip/asterisk/chan-lantiq) for the dialplan.

@majuss would you mind posting your configuration?

**Comment by majuss** [comment]
@ORippler sorry I'am not using the fixed line.

**Comment by ORippler** [comment]
Hmm okay. Based on the [OpenWRT Project](https://openwrt.org/toh/astoria/arcadyan_astoria_easybox_904xdsl_r01?s[]=arcadyan&s[]=astoria&s[]=easybox&s[]=904xdsl#installing_lede) page I just assumed it to be working...

Did anyone manage to successfully use the analog phone line yet?

**Comment by majuss** [comment]
The newest Snapshot should not boot at all. Did it booted for you!?

Use some older snapshot without _smp compiled in. This should work correctly with SIP but you wont get more than ~66 Mbit via DSL.

**Comment by ORippler** [comment]
Yeah I use the snapshot from 02/11 and installed all asterisk-related packages marked as compiled in the config via `opkg` from a pen-drive (funnily images i compile under fa8e9a88891841f6b4a35296315db68526cde2f1 with the same config provided with the snapshot don't boot or alternatively reboot every ~1min). Once #13 is fixed i expect snapshots to boot again.

I am currently struggling with parametrizing SIP, and was looking for help in that regard :)
Based on the fact that I can communicate with asterisk over the analog phone (e.g. Dialing a number resulsts in asterisk throwing exceptions) I guess that chan_lantiq works and I just didn'T get parameters correctly yet
